algorithms/src/merkle_tree/masked_merkle_tree_parameters.rs

```diff
@@ -37,16 +50,22 @@ impl<H: CRH, const DEPTH: usize> MerkleParameters for MaskedMerkleTreeParameters
     //  If you are seeing this message, please be proactive in bringing it up :)
     /// Returns the saved `message` from calling the `MerkleParameters::setup()` function.
     fn setup_message(&self) -> &str {
-        &self.2
+        &self.3
     }
 
-    fn crh(&self) -> &Self::H {
+    fn crh(&self) -> &Self::LeafCRH {
```

Comment:
nit: Rename to `leaf_crh` so it is clearly distinct from the `two_to_one_crh` (you should be able to do an `fn + F6`)